doc/api/stream.md

```diff
@@ -2816,6 +2816,8 @@ added: v17.0.0
 > Stability: 1 - Experimental
 
 * `streamReadable` {stream.Readable}
+* `options` {Object}
+  * `strategy` {QueuingStrategy}
```

Comment:
Is this `QueuingStrategy` referred to/documented elsewhere (is it a real type)?

Reply:
I was looking for help a hand here, since `QueuingStrategy` is not a type in the `stream.js` source for the documentation, but it's a type in the `webstream.js`. 

What is the workaround when you need a type that's not defined in the source file for the documentation?

Reply:
I'm not familiar enough with web streams, but if nothing else I guess we could just use `Object`.